packages/data-service/src/data-service.ts

```diff
@@ -2570,7 +2573,11 @@ export class DataServiceImpl extends EventEmitter implements DataService {
     return result;
   }
 
-  _getClientEncryption(): ClientEncryptionType {
+  _getClientEncryption(): any {
+    if (!ClientEncryption) {
+      return;
```

Comment:
Can we throw an exception instead? That’s almost certainly always better than returning `undefined` and letting the caller crash.